Alcancia-io/Alcancia-FrontEnd
# Fix: zendesk logout

## Summary
Fix uncaught exception when `ZendeskMessaging.logoutUser()` was called on user profile screen `dispose()` method if the no zendesk user was loggedIn .
This error did not affect functionality or UX in any way, it was just a minor annoyance while debugging.

## Requirements
_list any dependencies necessary to review the changes_
- _eg. make sure you reaplace your `.env` with the latest version_

## Type of change

## How to test?
Run the app as you usually do, go to User Profile tab and then go to any other tab, the uncaught exception should not appear on the console.

## How has this been tested?
Same as previous section

## Screenshots
N/A

## Checklist (for the reviewer)
